docs/reference/cluster/stats.asciidoc

```diff
@@ -34,7 +34,10 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=node-filter]
 [[cluster-stats-api-query-params]]
 ==== {api-query-parms-title}
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
+(Optional, <<time-units, time units>>) Time period to wait for
+a response. If no response is received before the timeout expires, the request
+fails and returns an error. Defaults to no timeout.
```

Comment:
I'd like to get confirmation on this default. I had a hard time tracing/parsing it:

https://github.com/elastic/elasticsearch/blob/0b536a7a71d97bc2b55c9f7c007fd4fdc162bac2/server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestClusterStatsAction.java#L50

Goes to:

https://github.com/elastic/elasticsearch/blob/41b2b2ee9004bce68ab61afc767b4eaa771d4a1e/server/src/main/java/org/elasticsearch/action/admin/cluster/stats/ClusterStatsRequest.java#L31

Goes to:

https://github.com/elastic/elasticsearch/blob/41b2b2ee9004bce68ab61afc767b4eaa771d4a1e/server/src/main/java/org/elasticsearch/action/support/nodes/BaseNodesRequest.java#L91

I interpreted this as no default timeout, but it's very possible I'm incorrect.

Reply:
Meant to tag @jbaiera ^

Reply:
I believe you are right.

The timeout here is for getting a stats response from the nodes in the cluster. It should respond 200 OK also when some of the nodes time out.